main.py

```diff
@@ -19,7 +19,7 @@
 #
 #enveloppe = img.correct_intensity_envelop()
 
-stitch = Stitching(sourceDir=sourcePath, tileD=tileDimensions, imageSize=[1024,512], isIntensityCorrection=True, isMirrored=True, isFlipped=False)
+stitch = Stitching(sourceDir=sourcePath, tileD=tileDimensions, imageSize=[1024,512], isIntensityCorrection=False, isMirrored=True, isFlipped=False)
```

Comment:
Split the loading of images and the stitching of images in two different functions.